### PR TITLE
Added octal to binary conversion

### PR DESCRIPTION
**Pull request by vishnu0pothan**
#### Checklist

- [x] Added description of change
- [x] Added file name matches [File name guidelines](https://github.com/TheAlgorithms/C/blob/master/CONTRIBUTING.md#File-Name-guidelines)
- [ ] Added tests and example, test must pass
- [ ] Relevant documentation/comments is changed or added
- [x] PR title follows semantic [commit guidelines](https://github.com/TheAlgorithms/C/blob/master/CONTRIBUTING.md#Commit-Guidelines)
- [x] Search previous suggestions before making a new one, as yours may be a duplicate.
- [x] I acknowledge that all my contributions will be made under the project's license.

Notes: 

<a href="https://gitpod.io/#https://github.com/TheAlgorithms/C/pull/629"></a>

### CODE REVIEW

File: conversions/octal_to_binary.c [hunk 1]
```diff
@@ -0,0 +1,54 @@
+/**
+ * @brief Octal to binay conversion by scanning user input
+ * @details
+ * The octalTobinary function take the octal number as long  
+ * return a long binary nuber after conversion
+ * @author Vishnu P(https://github.com/vishnu0pothan)
```
Comment (Panquesito7) [inline conversions/octal_to_binary.c:6]:
```suggestion
 * @author [Vishnu P](https://github.com/vishnu0pothan)
```

File: conversions/octal_to_binary.c [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+    {
+	binarynum = binarynum + (decimalnum % 2) * i;
+	decimalnum = decimalnum / 2;
+	i = i * 10;
+    }
+
+    //Returning the binary number that we got from octal number
+    return binarynum;
+}
+int main()
```
Comment (Panquesito7) [inline conversions/octal_to_binary.c:43]:
```suggestion

/**
 * @brief Main function
 * @returns 0 on exit
 */
int main()
```

File: conversions/octal_to_binary.c [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+ * @brief Octal to binay conversion by scanning user input
+ * @details
+ * The octalTobinary function take the octal number as long  
+ * return a long binary nuber after conversion
+ * @author Vishnu P(https://github.com/vishnu0pothan)
+ */
+#include <stdio.h>
+#include <math.h>
+
+//This function converts octal number to binary number
```
Comment (Panquesito7) [inline conversions/octal_to_binary.c:11]: A function should be documented like:

```c++
/**
 * @brief description of what the function does
 * @param <parameter name> description
 * @returns description
 */
```

**Review by Panquesito7 — CHANGES_REQUESTED** [review]

**Comment by vishnu0pothan** [comment]
Updated the function description. Changes made.

**Review by Panquesito7 — CHANGES_REQUESTED** [review]
Please enable [GitHub Actions](https://docs.github.com/en/free-pro-team@latest/actions/learn-github-actions) in your repository of this fork in this link: https://github.com/vishnu0pothan/C/actions

**Comment by vishnu0pothan** [comment]
Enabled the Actions

**Comment by Panquesito7** [comment]
Awesome, good work. To trigger the GitHub Actions, make a small commit. For example, remove line 61.
Next, we will see a commit made by the GitHub Actions updating `DIRECTORY.md`. 🙂

**Comment by vishnu0pothan** [comment]
Done @Panquesito7 .

**Review by Panquesito7 — APPROVED** [review]
LGTM, good work. 😄 👍

**Comment by Panquesito7** [comment]
> This pull request **introduces 1 alert** when merging [d57edeb](https://github.com/TheAlgorithms/C/commit/d57edebc47fb596866c801b9b3ef750fa80b982f) into [d3bb124](https://github.com/TheAlgorithms/C/commit/d3bb124b90f3f7aaa404ea1e59d21d6e667d64b7) - [view on LGTM.com](https://lgtm.com/projects/g/TheAlgorithms/C/rev/pr-13cbbce10bcdc18a257ea5cff82a87c058f226eb)
> 
> **new alerts:**
> 
> * 1 for Multiplication result converted to larger type

Please fix the [warning](https://lgtm.com/projects/g/TheAlgorithms/C/rev/pr-13cbbce10bcdc18a257ea5cff82a87c058f226eb) reported by LGTM.

**Pull request merged**